docs/docs-content/clusters/data-center/nutanix/register-nutanix-cloud.md

```diff
@@ -0,0 +1,246 @@
+---
+sidebar_label: "Register Nutanix Cloud"
+title: "Register Nutanix Cloud"
+description: "Learn how to register a Nutanix cloud in Palette."
+hide_table_of_contents: false
+sidebar_position: 5
+tags: ["data center", "nutanix"]
+---
+
+
+A system administrator registers the Nutanix cloud in Palette by invoking system-level APIs. These APIs provide specific cloud information, the cloud logo, and the key-value pairs required to add the cloud to Palette. They also enable uploading the YAML templates used to create the cluster, control plane, and worker nodes. This section provides instructions on how to download and modify the YAML templates, as well as how to use the APIs to register a Nutanix cloud to Palette.
+
+
+## Prerequisites
+
+
+- A Palette account with system console access. The user with this privilege is the *admin user* of the self-hosted instance of [Palette](https://docs.spectrocloud.com/enterprise-version/system-management/#system-console) or [VerteX](https://docs.spectrocloud.com/vertex/system-management/#system-console).
+
+- [`curl`](https://curl.se/docs/install.html) command installed.
+
+<!-- - A valid Palette authentication token. To learn how to acquire an authentication token, review the [Authorization Token](https://docs.spectrocloud.com/user-management/authentication/authorization-token) guide. -->
+
+
+## Setup
+
+Use the following steps to prepare to register your cloud with Palette. 
+
+### Customize YAML Configuration Files
+
+1.  Download the following YAML files from the [Nutanix Cluster API (CAPI) Provider](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix) GitHub repository:
+    - **infrastructure-components.yaml**
+    - **cluster-template.yaml**
+  
+  <br />
+
+  :::caution
+  Review the [Nutanix](https://opendocs.nutanix.com/capx/v1.2.x/validated_integrations/#validated-versions) compatibility matrix to ensure you download the latest CAPI version of the files. 
```

Comment:
We should have a requirement for the Nutanix CAPI version required, that must be compatible with our CAPI version.

Reply:
Added the second bullet, following the existing bullet about minimum supported CAPI version 1.5.3.

"- A Nutanix cloud created with the minimum supported CAPI version 1.5.3. Refer to the Nutanix [Validated Integrations](https://opendocs.nutanix.com/capx/v1.2.x/validated_integrations/#validated-versions) compatibility matrix.

- The Nutanix CAPI version must be compatibe with Palette's CAPI version."